build.gradle

```diff
@@ -59,4 +61,62 @@ dependencies {
 test {
     jvmArgs("-XX:+EnableDynamicAgentLoading", "-Xshare:off")
     useJUnitPlatform()
-}
\ No newline at end of file
+}
+
+node {
+    download = true
+    version = '18.16.1'
+    npmVersion = '9.5.1'
+    // when setting both these directories, npm and node will be in separate directories
+    workDir = file("${buildDir}/nodejs")
+    npmWorkDir = file("${buildDir}/npm")
+}
+
+def npmExec = System.getProperty('os.name').toLowerCase().contains('windows') ? '/npm.cmd' : '/bin/npm'
+def nodeExec = System.getProperty('os.name').toLowerCase().contains('windows') ? '/node.exe' : '/bin/node'
+
+spotless {
+    format 'misc', {
+        // define the files to apply `misc` to
+        target '*.gradle', '.gitattributes', '.gitignore'
+
+        // define the steps to apply to those files
+        trimTrailingWhitespace()
+        indentWithSpaces(4)
```

Comment:
Consider changing to 2, as this seems to be standard of some industry linters? Need to discuss this.